### PR TITLE
[zh-cn] Resync stateful application page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh-cn/docs/tasks/run-application/run-replicated-stateful-application.md [hunk 1]
```diff
@@ -124,11 +126,11 @@ ConfigMap 本身没有什么特别之处，因而也不会出现不同部分应
 每个 Pod 都会在初始化时基于 StatefulSet 控制器提供的信息决定要查看的部分。
 
 <!--
-### Services 
+### Create Services {#services}
 
 Create the Services from the following YAML configuration file: 
 -->
-### 服务  {#services}
+### 创建服务  {#services}
```
Comment (sftim) [inline content/zh-cn/docs/tasks/run-application/run-replicated-stateful-application.md:133]: Would it help to write “Service” here (the name of the API kind that the reader must create)? The capital S in Services is a hint there that the page is referring to the API not the English word.

File: content/zh-cn/docs/tasks/run-application/run-replicated-stateful-application.md [hunk 2]
```diff
@@ -346,17 +356,16 @@ MySQL 本身不提供执行此操作的机制，因此本示例使用了一种
...
 [sidecar](https://kubernetes.io/blog/2015/06/the-distributed-system-toolkit-patterns). 
 -->
 ### 开始复制
 
-Init 容器成功完成后，应用容器将运行。
-MySQL Pod 由运行实际 `mysqld` 服务的 `mysql` 容器和充当
-[辅助工具](https://kubernetes.io/blog/2015/06/the-distributed-system-toolkit-patterns)
-的 xtrabackup 容器组成。
+初始化容器成功完成后，应用容器将运行。MySQL Pod 由运行实际 `mysqld` 服务的 `mysql`
+容器和充当[辅助工具](https://kubernetes.io/blog/2015/06/the-distributed-system-toolkit-patterns)的
```
Comment (sftim) [inline content/zh-cn/docs/tasks/run-application/run-replicated-stateful-application.md:367]: Also OK IMO:
```suggestion
容器和充当[辅助工具](/blog/2015/06/the-distributed-system-toolkit-patterns/)的
```

**Review by sftim — COMMENTED** [review]
I spotted some mini-nits, possibly.

Optional extra change (near the end of the file):
```diff
-* 进一步了解[强制删除 StatefulSet Pods](/zh-cn/docs/tasks/run-application/force-delete-stateful-set-pod/).
+* 进一步了解[强制删除 StatefulSet Pod](/zh-cn/docs/tasks/run-application/force-delete-stateful-set-pod/).
```

**Review by howieyuen — APPROVED** [review]
/approve

File: content/zh-cn/docs/tasks/run-application/run-replicated-stateful-application.md [hunk 3]
```diff
@@ -91,11 +91,13 @@ and a StatefulSet.
...
-### ConfigMap
-
 <!--
+### Create a ConfigMap {#configmap}
+
 Create the ConfigMap from the following YAML configuration file:
 -->
+### 创建一个 ConfigMap   {#configmap}
+
 使用以下的 YAML 配置文件创建 ConfigMap ：
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/run-application/run-replicated-stateful-application.md:101]:
```suggestion
使用以下的 YAML 配置文件创建 ConfigMap：
```

**Review by Sea-n — COMMENTED** [review]
Non-blocking nitpick. Looks good to me, it's ready to merge.
/lgtm

**Pull request merged**